src/slackmessageparser.ts

```diff
@@ -4,7 +4,10 @@ import * as Markdown from "markdown-it";
 import * as MarkdownSlack from "markdown-it-slack";
 import * as escapeHtml from "escape-html";
 
-const md = Markdown();
+const md = require('markdown-it')({
```

Comment:
try to avoid `require`, to keep things more typescript-y
```
import * as MarkdownIt from "markdown-id";
const md = MarkdownIt({
    breaks: true,
});
```

Reply:
I think I tried it like this, but it failed:

```
const md = Markdown({
    breaks: true,
});
```

I'll give it another look